src/components/Button/ButtonInputProps.ts

```diff
@@ -3,6 +3,9 @@
  * @author Lia Arroyo <liayzabel@gmail.com>
  */
 type ButtonInputProps = {
+  className?: string;
```

Comment:
```suggestion
  className?: string | null;
```

since it can be null?